integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/generator.py

```diff
@@ -142,7 +142,8 @@ def __init__(
         if max_length is not None or truncate is not None:
             warnings.warn(
                 "The 'max_length' and 'truncate' parameters have been removed and no longer have any effect. "
-                "No truncation will be performed.",
+                "No truncation will be performed. To control the length of the generated text, use "
+                "the 'max_tokens' (or the model-specific variant) in the generation_kwargs of the run method.",
```

Comment:
I'd say which models accept `max_tokens` here and perhaps (if easily possible) direct users to where they could find out what the correct model-specific variant is.

Reply:
Perhaps we could add to the kwargs param in the init docstrings where users could look to find these model specific kwargs?